src/Microsoft.OData.Core/Json/IJsonWriterFactory.cs

```diff
@@ -18,9 +19,10 @@ public interface IJsonWriterFactory
         /// <summary>
         /// Creates a new JSON writer of <see cref="IJsonWriter"/>.
         /// </summary>
-        /// <param name="textWriter">Writer to which text needs to be written.</param>
+        /// <param name="stream">The stream write to.</param>

```

Comment:
why `the stream write to` and not `the stream to write to? `

Reply:
@ElizabethOkerio Fixed. Good catch! It should be "The stream to write to". The error was replicated because I tend to copy the param docstring across different places where the param represents the same thing.